pkg/env/test.go

```diff
@@ -96,3 +96,12 @@ func (t *Test) ACRResourceID() string {
 func (t *Test) ACRName() string {
 	return "arosvc"
 }
+
+// https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azurite
+func (t *Test) E2EStorageAccountName() string {
+	return "devstoreaccount1"
+}
+
+func (t *Test) E2EStorageAccountKey() string {
+	return "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="
```

Comment:
Even though this is for a fake storage account. It feels weird to have this in code. If we move to a mock I'd like this to be an empty string or "accountKey" or something of that sort.  And how is this static.  Wouldn't azurite need to accept this or have it passed in during creation of the container/storage account?

Reply:
Test env is purely for unittests

Reply:
they were when i was using `azurite` (storage emulator) for UTs but since i removed it i'll also remove this.